examples/ex1p.cpp

```diff
@@ -88,6 +98,19 @@ int main(int argc, char *argv[])
    Mesh *mesh = new Mesh(mesh_file, 1, 1);
    int dim = mesh->Dimension();
 
+   if ( dim == 2 && nz > 0 )
```

Comment:
This is cool, but I'm on the fence if the complication in the example is worth it... If we decide to keep it, should we update the serial `ex1.cpp` to match?

Reply:
I understand your concern.  We could make a new example which demonstrates mesh extrusion either here or in miniapps/meshing.

Reply:
Maybe this could be an option in `mesh-explorer`?

Reply:
It could but frankly I'd like to show it in a dedicated example so that users could see just how simple it is.  The user interface in mesh-explorer can obscure the operations being performed.